task_1.js

```diff
@@ -0,0 +1 @@
+console.log("Panda");
```

Comment:
```suggestion
console.log('Panda');
```
Please use single quote.